Project.toml

```diff
@@ -4,7 +4,9 @@ version = "0.5.0"
 
 [deps]
 AbstractMCMC = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
+DiffResults = "163ba53b-c6d8-5494-b064-1a9d43ac40c5"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
```

Comment:
Please consider loading `ForwardDiff` and `DiffResults` optionally using `Requires.jl`.

Reply:
Hi. Re: HMC, yes I'm with you. However, in my experience the special case of HMC you are describing seemed to run a bit slowly (more than I might expect). I wonder'd if, with a dedicated MALA implementation it could be easier to optimize, and also, easier for the user.

Reply:
Makes sense!

Reply:
Happy to use Requires.jl, but the documentation is not super clear. Can you describe (briefly) how you think I might use it (wrt the two packages)? Thanks.

Reply:
I guess you should nest the optional requirements, as shown in https://github.com/JuliaPackaging/Requires.jl#multiple-requirements. Does that answer your question?